charts/orchestrator/templates/rhdh-operator.yaml

```diff
@@ -16,9 +16,6 @@ spec:
         - name: app-config-rhdh-catalog
     dynamicPluginsConfigMapName: dynamic-plugins-rhdh
     extraEnvs:
-      envs:
-      - name: NODE_TLS_REJECT_UNAUTHORIZED
-        value: "0"
```

Comment:
is this a change related to this pr?

Reply:
it isn't part of the current version of the PR.
was added accidently, however, removed, but since I was a bit lazy to do it from github directly, i haven't squashed the commits :-P 
going to do it now